lib/workers/repository/onboarding/branch/check.ts

```diff
@@ -138,7 +140,7 @@ export async function isOnboarded(config: RenovateConfig): Promise<boolean> {
     await ensureComment({
       number: pr.number,
       topic: `Renovate is disabled`,
-      content: `Renovate is disabled due to lack of config. If you wish to reenable it, you can either (a) commit a config file to your base branch, or (b) rename this closed PR to trigger a replacement onboarding PR.`,
+      content: `Renovate is disabled due to lack of config. If you wish to re-enable it, you can either (a) commit a config file to your base branch, or (b) rename this closed PR to trigger a replacement onboarding PR.`,
```

Comment:
Shouldn't we allow a force rebase via checkbox?

Reply:
This doesn't seem related to this PR?

Reply:
🤔 Can't we currently use the PR checkbox to force a wipe and new commit for the onbarding branch and we loose that with this change? 🤔

Reply:
I just mean that this PR only added a hyphen to the text, nothing else?

Reply:
Acc to new logic a rebase has no meaning because we merge updated base into onboarding on each run.